crates/routes/src/images/delete.rs

```diff
@@ -138,3 +139,19 @@ pub async fn delete_image(
 
   Ok(Json(SuccessResponse::default()))
 }
+
+/// Deletes any image, only for admins.
+pub async fn delete_image_admin(
+  data: Json<DeleteImageParams>,
+  context: Data<LemmyContext>,
+  local_user_view: LocalUserView,
+) -> LemmyResult<Json<SuccessResponse>> {
+  is_admin(&local_user_view)?;
+
+  LocalImage::delete_by_alias(&mut context.pool(), &data.filename).await?;
+
+  // Use purge, since it should remove any other aliases.
+  purge_image_from_pictrs(&data.filename, &context).await?;
```

Comment:
Agree with using purge here, but then we should also delete other aliases for the same image from `LocalImage` table. Pict-rs does return an array `aliases` from the purge endpoint.

Reply:
I think that's impossible, as we don't store anything else. I don't even think pictrs exposes real ids.

Reply:
Thx, got it.